api/plugin/file/filehost.go

```diff
@@ -106,26 +107,16 @@ func (*FilePlugin) Init(ctx context.Context, e eng.Engine) bool {
 }
 
 func (fi *fileInfo) Read(p []byte) (n int, err error) {
-	n, err = fi.rdClose.Read(p)
-	if err == io.EOF {
-		log.Printf("We read %d bytes and the file has exhausted its content", n)
-	} else if err != nil {
-		log.Fatal("Error reading from a file: ", err)
-	} else {
-		log.Printf("We read %d bytes", n)
-	}
-
-	return n, err
+	return fi.rdClose.Read(p)
 }
 
 func (fi *fileInfo) Write(p []byte) (n int, err error) {
 	n, err = fi.wrClose.Write(p)
 	if err != nil {
-		log.Fatal("Error writing to a file: ", err)
+		return 0, err
 	}
 
 	fi.size += n
```

Comment:
Is `size` here mean "the current place to write"?

Reply:
not exactly. Actually, it's the byte size of the existing file content. but in the situation of writing, it could also be seen as the "current place to write"